content/guides/dashboard/github-integration.md

```diff
@@ -74,20 +91,102 @@ to GitHub.com to complete the installation:
 
 3. Click the **Install** button to complete the installation.
 
+### Cypress GitHub Enterprise app installation process
+
+To integrate the Dashboard with GitHub Enterprise, you need to create a new
+GitHub App, copy the necessary settings and credentials from the new app, paste
+them into the Dashboard, and complete the activation process.
+
+1. With your Dashboard organization's GitHub Enterprise page open, open a new
+   tab or browser window and browse to your GitHub Enterprise site. Navigate to
+   your GitHub organization → **Settings** → **Developer settings** → **GitHub
+   Apps**. Do **NOT** go to **OAuth Apps**. Click the **New GitHub App** button.
+
+<DocsImage src="/img/dashboard/github-integration/ghe/ghe-01.png" alt="Create new GitHub App" id="cypress-github-enterprise-app-installation-process-1"></DocsImage>
+
+2. Complete the **Register new GitHub app** section.
+   - Enter a **GitHub App name**. Name may contain only dashes, letters and
+     numbers, **no spaces**.
+   - Enter the Cypress Dashboard **Homepage URL**, https://dashboard.cypress.io
+   - Enter the **Callback URL** and **Setup URL**,
+     https://dashboard.cypress.io/apps/github/callback
+
+<DocsImage src="/img/dashboard/github-integration/ghe/ghe-02.png" alt="Configure new GitHub App" id="cypress-github-enterprise-app-installation-process-2"></DocsImage>
+
+3. Complete the **Webhook** section.
+   - Enter the **Webhook URL**, https://dashboard.cypress.io/webhooks/github-app
+   - Copy and paste the provided **Webhook secret** from your Cypress Dashboard
+     to **Webhook secret (optional)**.
+
+<DocsImage src="/img/dashboard/github-integration/ghe/ghe-03.png" alt="Configure app webhook" id="cypress-github-enterprise-app-installation-process-3"></DocsImage>
+
+4. Set the **Repository Permissions**. Below are the minimum permissions
+   required for the new GitHub App.
+   - **Actions**: read-only
+   - **Checks**: read & write
+   - **Contents**: read-only
+   - **Discussions**: read & write
+   - **Pull requests**: read & write
+   - **Commit statuses**: read & write
+
+<DocsImage src="/img/dashboard/github-integration/ghe/ghe-04.png" alt="Configure app permissions" id="cypress-github-enterprise-app-installation-process-4"></DocsImage>
+
+5. Skip to the bottom of the form and click the **Create GitHub App** button.
+   Your new GitHub App is now created, and you'll be taken to the settings page.
+
+6. Returning to the Cypress Dashboard GitHub Enterprise Integration
+   configuration screen, populate the following values from your new GitHub App
+   - Copy the root URL of your GitHub Enterprise site to the Cypress **GitHub
+     Enterprise Server Url** field.
+   - Copy the **App ID** to the Cypress **GitHub Enterprise App ID** field.
+   - Copy the **Client ID** to the Cypress **GitHub Enterprise Client ID**
+     field.
+   - Copy the **GitHub App name** to the Cypress **GitHub Enterprise app name**
```

Comment:
Still need to include a screenshot of the GitHub App name field from GitHub

Reply:
It's already there https://cypress-documentation-n6c1j1liv-cypress-io.vercel.app/guides/dashboard/github-integration#cypress-github-enterprise-app-installation-process-5

Reply:
See it now. Was expecting the field, not the header, but that works. Will resolve.